src/components/RainbowButton.jsx

```diff
@@ -0,0 +1,46 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { Button, ButtonProps } from '@sumup/circuit-ui';
+
+const StyledButton = styled(Button)(
+	() => css`
```

Comment:
🤔 is the foundry setup not switching these table to spaces? Doesn't matter much, but that's strange